### DISCUSSION TITLE
wire a moes MS-105z dimmer with regular on off switch

### DISCUSSION THREAD

**Discussion by hocomadvies** [Koenkk/zigbee2mqtt#14627] [Devices]
Since we do not have a real forum for questions like this I just give i a try here :) I'm sure I'm not the only person with questions like this :) 

Although it seems to work I have some doubts if I did the wiring for the MS-105z the right way. I have it connected to zigbee2mqtt A device is created. And I can control it. But the on / off using the switch / or controller made me wondering if I have it plugged in the right way. Since it starts circling through brightens up and down and only stops when the switch is turned off. Is it possible just use the switch to have the light switch on or of at a certain default brightness? And when needed have it turn up or down (by app or switch)? (I'm using the Dutch wiring). 

<img width="1764"  src="https://user-images.githubusercontent.com/10847395/197512964-12d6d338-f50a-4fc0-bac5-0f07f107984b.png">

![IMG_4705](https://user-images.githubusercontent.com/10847395/197513214-f529dfb3-b599-458d-8c80-65cad837629e.jpg)

**Comment by davew21** [comment]
As far as I know you can’t, the switch has to be a momentary type so when released the contacts open again, more like a push button.

**Reply by hocomadvies** [reply]
Well it seems it is working with the normal switch. A quick on - off from the switch makes it behave as needed and switches the light off. When just one time switch it will cycle through the brightness. Than switching it one more time will make it stay on the selected brightness.

**Comment by mararn1618** [comment]
Hey @hocomadvies , thanks for your post, I have just installed two MOES dimmers and experience the same behaviour. Quick on-Off workaround works for me too. It seems the Dimmers are rather made for push buttons than switches. Did you find a solution?
